easybuild/easyconfigs/s/spglib/spglib-1.14.1-intel-2019a.eb

```diff
@@ -0,0 +1,26 @@
+easyblock = 'CMakeMake'
+name = 'spglib'
+version = '1.14.1'
+
+homepage = 'https://atztogo.github.io/spglib/'
+description = """Spglib is a library for finding and handling crystal symmetries written in C."""
+
+toolchain = {'name': 'intel', 'version': '2019a'}
+
+builddependencies = [
+    ('CMake', '3.13.3'),
+]
+
+source_urls = ['https://github.com/atztogo/spglib/archive/']
+sources = ['v%(version)s.tar.gz']
```

Comment:
missing checksum (tip: you can use the eb options `--check-contrib` to check, `--inject-checksums` to add checksums and `--update-pr` to update this PR)